src/ixdat/techniques/ec_ms.py

```diff
@@ -154,6 +155,11 @@ def ecms_calibration_curve(
                 Defaults to a new axis.
             axes_measurement (list of Axes): The EC-MS plot axes to highlight the
                 ms_calibration on. Defaults to None. These axes are not returned.
+            axes_measurement_J_name (str): The J_name used in the axis passed 
```

Comment:
okay, but wouldn't it be more flexible to accept a J_unit_plot and then decide whether to use raw or normalized current based on whether it had a /cm^2 in it?
Maybe worth a revisit once ixdat has real units.